src/content/terraform/product-landing.json

```
@@ -1,20 +1,20 @@
 {
 	"hero": {
-		"heading": "Terraform Placeholder Duplicate Of Vault Content",
+		"heading": "Automate Infrastructure on Any Cloud",
 		"image": "https://place-hold.it/600x320"
 	},
 	"overview": {
 		"heading": "What is Terraform?",
-		"body": "Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.",
+		"body": "Terraform is an infrastructure as code tool that lets you build, change, and version infrastructure safely and efficiently. This includes low-level components like compute instances, storage, and networking; and high-level components like DNS entries and SaaS features.",
```

```suggestion
		"body": "Terraform is an infrastructure as code tool that lets you build, change, and version infrastructure safely and efficiently. This includes low-level components like compute instances, storage, and networking; and high-level components like DNS entries and SaaS offerings.",
```

Features feels too specific, narrow-scoped. But maybe my brain is broken today.

---

I think the idea is that Terraform can control features of SaaS offerings, but not the offering itself. For example, you can create GitHub orgs and teams using Terraform, but you can't sign up for GitHub.

---

Yep, my brain is broken today. What you said makes sense.